### ISSUE TITLE
Question about the pre-trained weights

### ISSUE THREAD

**Issue by fb82** [xuy123456/fcgnn#3]
Hi again,

are the pre-trained weights computed by using also MegaDepth and ScanNet image datasets? If so, are the training data distinct from the test sets used for instance in the LoFTr and AspanFormer?

Thanks!
Fabio

**Comment by xuy123456** [comment]
Our network is trained on the MegaDepth dataset, and scenes in the test set are pre-excluded during training.

**Comment by fb82** [comment]
Got it, thanks!
